packages/spatial/src/physics/classes/Physics.ts

```diff
@@ -293,6 +293,10 @@ const setRigidBodyType = (world: PhysicsWorld, entity: Entity, type: Body) => {
   }
 
   rigidbody.setBodyType(typeEnum, false)
```

Comment:
Have you tri3d waking the body up by changing this line's second parameter to true?

Reply:
yes, didn't work

Reply:
https://github.com/pmndrs/react-three-rapier/issues/314